### ISSUE TITLE
Code Quality: Function Naming

### ISSUE THREAD

**Issue by JoeDailey** [motocru/curse-bot#7]
This is overall feedback. You should decided what it means to close this issue. (This includes doing nothing.)
---

I've noticed a theme in your function naming. Each is given an *object-like* name. It's natural to get caught up doing this, but its a habit to avoid. 

https://github.com/motocru/curse-bot/blob/6d870abee030ef900986f101712449f45ce3bea5/bot.js#L226

These are not *objects*, they are *functions*. From a **caller**-perspective, it's more important to talk about *what* they do than *who* they are. When you only give me their 'name', I cannot predict what they'll do. This is incredibly important in JavaScript because functions do not have return types. 

Here an example:
My friend's name is **Pete**, he is really good at *something* that involves a **bat**. I'll call him *batPete*. One day I'm playing baseball so I call on `batPete()` and he kills my grandmother.

I hope this highlights the confusion from naming a function `messageDecider`.

## Recommended Fix
Function calls should express three* things:
- What it's operating on
- What it's going to do
- What it's going to return

So, for example, lets rename `messageDecider`.
```
function composeMessage(user, curse, difference, MILESTONES) {
  // I do not need to know the implementation because the 
  // name and inputs tell me everything...
  // > `composeMessage` with compose a message based on
  // > new milestones reached.
}
```

TL;DR > Functions are *verbs* not *nouns*.
---
*If you're writing a function that modifies *state*, your function should also express this. The functions in this project are mostly *functional*, which is a good thing!
